### PR TITLE
Post v9.0 release IDD changes and related doc cleanup

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
IDD changes to drop in after v9.0 release.

1. The object-list for Defrost Energy Input Ratio Modifier Function of Temperature Curve Name for `AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl*` should be BivariateFunctions (not Univariate). See #6948.

2. `AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl:HR` has two fields with the index N43. And fix inconsistent case for \type alpha. See #6815.

3. `Coil:Cooling:Water` Heat Exchanger Configuration idd default CounterFlow not consistent with documentation. Fixed documentation. See #5601.

4. Invalid key is set as default for Season input field in `UtilityCost:Charge:Block` object. Default should be Annual. See #6904.

5. `Fan:SystemModel` not included in all applicable reference lists, including valid branch components. See #7013.

6. Evap cooler idd note typo. See [comment below](https://github.com/NREL/EnergyPlus/pull/6951#issuecomment-431368245).

7. Incorrect object name referenced in `ZoneAirBalance:OutdoorAir` Name field. See [comment below](https://github.com/NREL/EnergyPlus/pull/6951#issuecomment-433171975)

8. Remove idd object for `FaultModel:PressureSensorOffset:OutdoorAir`, this object is not implemented, and is [not present in the documentation](https://github.com/NREL/EnergyPlus/issues/6717#issuecomment-429385401).

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
 - [ ] Required documentation updates?

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@mjwitte @lgentile it has been 14 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
@mjwitte you are welcome to drop this in anytime you are ready.

**Comment by jmarrec** [comment]
@mjwitte while you're changing the IDD there's a typo on EvaporativeCooler:Direct:ResearchSpecial

https://github.com/jmarrec/EnergyPlus/blob/develop/idd/Energy+.idd.in#L57817

       \note x = ff = flow fraction on the primary air. The flow fraction is the secondary air flow rate during current operation divided

replace "secondary" with "primary"

**Comment by rraustad** [comment]
Also a mistake in IO Ref ZoneAirBalance:OutdoorAir pg 591. Looks like object name got changed during design/develop and this text wasn't updated.

1.16.6.1.1 Field: Name
The name of the ZoneOutdoorAir:Combined object.

**Comment by mjwitte** [comment]
Calling this finished, ready for review after CI confirms.

File: doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex [hunk 1]
```diff
@@ -280,7 +280,7 @@ \subsubsection{Operating Block Calculations:}\label{operating-block-calculations
 
 \subsubsection{Effectiveness Equations}\label{effectiveness-equations}
 
-There are two modes of flow for the coil, Counter Flow mode or the Cross Flow mode, default set up is as cross flow since most air condition applications have cross flow heat exchangers. According to the mode of flow the following NTU - Effectiveness relationships are used to calculate coil effectiveness, which is used later by all the three modes (Dry, Wet, Part Wet) for calculating air outlet conditions and heat transfer.
+There are two modes of flow for the coil, CounterFlow or CrossFlow. The default mode is CounterFlow. According to the mode of flow the following NTU - Effectiveness relationships are used to calculate coil effectiveness, which is used later by all the three modes (Dry, Wet, Part Wet) for calculating air outlet conditions and heat transfer.
```
Comment (Myoldmopar) [inline doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex:283]: :+1:

File: idd/Energy+.idd.in [hunk 2]
```diff
@@ -20038,7 +20038,7 @@ RoomAirSettings:UnderFloorAirDistributionExterior,
 RoomAir:Node:AirflowNetwork,
       \memo define an air node for some types of nodal air models
   A1, \field Name
-      \type Alpha
+      \type alpha
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:20041]: :+1:

File: idd/Energy+.idd.in [hunk 3]
```diff
@@ -61443,7 +61449,7 @@ AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl,
        \note OnDemand to defrost only when necessary
   A10, \field Defrost Energy Input Ratio Modifier Function of Temperature Curve Name
        \type object-list
-       \object-list UnivariateFunctions
+       \object-list BivariateFunctions
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:61452]: OK, so everything was implemented properly in the code, but IDF-Editor wouldn't show the right curves.  Got it.

File: idd/Energy+.idd.in [hunk 4]
```diff
@@ -61905,7 +61911,7 @@ AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl:HR,
        \note the steady-state value according to the inputs for
        \note Heat Recovery Heating Capacity Modifier and Heat Recovery
        \note Heating Capacity Time Constant
-  N43, \field Heat Recovery Heating Capacity Time Constant
+  N42, \field Heat Recovery Heating Capacity Time Constant
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:61914]: Yikes...

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
CI results look good, code changes look good.  Built locally, no issues. Ran tests, no issues.  Spot checked doc changes, looks good.  Merging.

**Pull request merged**